packages/dashboard/src/icons/svg-imports.d.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare module '*.svg' {
+  const content: string;
+  export default content;
+}
```

Comment:
Getting this when opening the Dashboard locally:

<img width="1131" alt="Screenshot 2023-01-06 at 15 13 59" src="https://user-images.githubusercontent.com/3294597/211073258-ad8ecec2-a207-401d-9d50-76321d8f286e.png">


Is there any dep missing from the packages now? Didn't see anything specific in the moveable nor in the `story-editor` package where this kind of import is done for inline svg-s. Thoughts? cc @swissspidy @barklund

Reply:
@miina Can you try changing the regex on these two lines:

https://github.com/GoogleForCreators/web-stories-wp/blob/2b8b788c961d3d6c0f040f9ba057a963b99e14b3/webpack.config.cjs#L133

https://github.com/GoogleForCreators/web-stories-wp/blob/2b8b788c961d3d6c0f040f9ba057a963b99e14b3/webpack.config.cjs#L163

If it works, need to make the same change for the storybook here:

https://github.com/GoogleForCreators/web-stories-wp/blob/2b8b788c961d3d6c0f040f9ba057a963b99e14b3/.storybook/main.cjs#L153

https://github.com/GoogleForCreators/web-stories-wp/blob/2b8b788c961d3d6c0f040f9ba057a963b99e14b3/.storybook/main.cjs#L183